PyRDF/Operation.py

```diff
@@ -88,42 +89,8 @@ def __init__(self, name, *args, **kwargs):
     def _classify_operation(self, name):
         # Classifies the given operation as action or
         # transformation and returns the type.
-
-        future_ops = [
-        "Range",
-        "Take",
-        "Snapshot",
-        "Foreach"
-        ]
-
-        if name in future_ops:
-            raise NotImplementedError("This operation ({}) isn't supported yet ! ".format(name))
-
-        ops = Operation.Types
```

Comment:
`Operation.Types`, `ops` and `operations_dict` should stay here, in `Operation`.